modules/jenkins-fortify-slave/main.tf

```diff
@@ -0,0 +1,112 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# THESE TEMPLATES REQUIRE TERRAFORM VERSION 0.8 AND ABOVE
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.9.3"
+}
+
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Create the Fortify Jenkins Node instance
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_instance" "jenkins_fortify_node" {
+  instance_type               = "${var.instance_type}"
+  ami                         = "${var.ami_id}"
+  key_name                    = "${var.ssh_key_name}"
+  subnet_id                   = "${var.subnet_ids[length(var.subnet_ids) - 1]}"
```

Comment:
since you are only launching a single instance, you can use a single string value for the subnet_id instead of a list of subnet_ids